### ISSUE TITLE
[Tech Debt] 526 Claims: Payee Code Errors

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#11353]
## Issue Description
when we submit a form to EVSS we sometimes get back the error:

form526.submit.establishClaim.serviceError: Benefit Type, Payee code or EP Code is not valid. Benefit Type must be CPD, Payee code must be 00, 10 - 29, 50, 60 or 70 - 78.

What can we do to better handle this error?

Note: This is not something we submit and does not feel like a retryable error so likely need to just tell EVSS to fix it.

## Tasks
- [x] Confirm this is still happening in prod
- [ ] (if still happening) Ask EVSS what this error means (provide them with some IDs)

## Acceptance Criteria
- [ ] If still happening, understand why and provide technical solution (code up if within estimate of 5 points)

**Comment by annaswims** [comment]
affected 49 users in the past month

**Comment by annaswims** [comment]
we can find instances of this by running `rake form526:errors` in prod.

**Comment by edmangimelli** [comment]
Waiting on Harlow's response (DM re how best to get EDIPI's to him)

**Comment by annaswims** [comment]
VA encrypted email is probably his preference, but the challenge is how to get it to citrix to send 😢

**Comment by edmangimelli** [comment]
Harlow responded. DM'ed EDIPI's

**Comment by edmangimelli** [comment]
Harlow update:

>No word yet. We opened SNOW ticket INC******** on 9/22 and today we emailed the VBMS ProdOps team to escalate the ticket.

**Comment by edmangimelli** [comment]
VBMS responded:
https://dsva.slack.com/archives/C8R3JS8BU/p1601480188000100?thread_ts=1600266051.043700&cid=C8R3JS8BU

**Comment by annaswims** [comment]
Silvio did some work recently on parsing participant id's. https://github.com/department-of-veterans-affairs/vets-api/pull/4104. I'd like to see if there isn't more we could do to determine which id is correct if there is a spouse id present.

**Comment by luke-gcio** [comment]
Would you say the only thing left for this story is front end work to improve the messaging?  Since we have the duplicate MVI thing already being worked on separately.

**Comment by annaswims** [comment]
SSOe might be able to help us get the correct id - https://dsva.slack.com/archives/CSFV4QTKN/p1605652868162100?thread_ts=1605646754.160800&cid=CSFV4QTKN

**Comment by luke-gcio** [comment]
Based on the comment from Anna we should use the SSOe rollout (and write a new ticket for this that captures the updated direction).

**Comment by edmangimelli** [comment]
About to push an updated PR [BIRLS PR](https://github.com/department-of-veterans-affairs/vets-api/pull/5245)

**Comment by edmangimelli** [comment]
My hope is that this [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/5245), when completely updated will help resolve this class of errors, as well as the errors mentioned in #13949

Unfortunately, [EVSS is revamping their error responses right now](https://dsva.slack.com/archives/C8R3JS8BU/p1611758637005300?thread_ts=1611703053.003700&cid=C8R3JS8BU) (less 500s, more 400s) which might make it hard to figure out the impact of my PR.

**Comment by edmangimelli** [comment]
Responding to feedback on this PR

**Comment by edmangimelli** [comment]
requested more feedback from Anna

**Comment by edmangimelli** [comment]
first PR merged
https://github.com/department-of-veterans-affairs/vets-api/pull/5924
pushing up the next

**Comment by edmangimelli** [comment]
still waiting on a VSP review of https://github.com/department-of-veterans-affairs/vets-api/pull/5991

**Comment by annaswims** [comment]
SSOe and retrying all possible BIRLS id's don't fix the error (at least not entirely)

**Comment by annaswims** [comment]
passed on another 88 vets who were affected, via va email.  I emailed Joe and Kimberly.  

I also made a `form526:corp_id_errors` rake task.  Ideally we shouldn't have to ask for this to be done manually, but that's what we've got for now.  We can regularly (weekly?) send this list to the people who can do something about it.
